examples/tutorials/traits_4.0/interfaces/interfaces.py

```diff
@@ -101,7 +101,7 @@ class Apartment(HasTraits):
 the specified class or one of its subclasses.
 """
 # --<Imports>------------------------------------------------------------------
-from traits.api import *
+from traits.api import HasTraits, Instance, implements, Interface, Str
```

Comment:
Same orthogonal observation. On master, `implements` is not defined.

Reply:
I've added #437 to the milestone for the 6.2 release.